tests/cloner_test.go

```diff
@@ -2786,10 +2715,11 @@ var _ = Describe("all clone tests", func() {
 					By("Check host assisted clone is taking place")
 					pvc, err := f.K8sClient.CoreV1().PersistentVolumeClaims(targetNs.Name).Get(context.TODO(), dvName, metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
-					suffix := "-host-assisted-source-pvc"
-					Expect(pvc.Annotations[controller.AnnCloneRequest]).To(HaveSuffix(suffix))
-					Expect(pvc.Spec.DataSource).To(BeNil())
-					Expect(pvc.Spec.DataSourceRef).To(BeNil())
+					if pvc.Spec.DataSourceRef == nil {
```

Comment:
Maybe we should go for a more correct way of ensuring dumbclone took place?
IIUC when this runs on a CSI lane the check is not exercised

Reply:
This is specifically for checking non-csi case.  With populators, the pvc that was used for dumb clone is deleted after rebinding to target.  But can check clonetype annotation on the dv